web_src/less/console/console.less

```diff
@@ -0,0 +1,295 @@
+// Based on https://github.com/buildkite/terminal-to-html/blob/697ff23bd8dc48b9d23f11f259f5256dae2455f0/assets/terminal.css
+
+.console {
+  background: #171717;
+  border-radius: 5px;
+  color: white;
```

Comment:
Can you variablize these fg/bg colors? E.g. `--color-console-fg`, `--color-console-bg`?

Reply:
done